cv19ResSupportV3/V4/Controllers/CallHandlersController.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using cv19ResSupportV3.V3.Controllers;
+using cv19ResSupportV3.V3.Domain.Commands;
+using cv19ResSupportV3.V4.UseCase.Interface;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using cv19ResSupportV3.V4.UseCase.Interfaces;
+
+namespace cv19ResSupportV3.V4.Controllers
+{
+    [ApiController]
+    [Route("api/v4/call-handlers")]
+    [Produces("application/json")]
+    // Check service api version information
+    [ApiVersion("3.0")]
```

Comment:
This is close to unimportant, but I think this might be the 4th version. From what I remember Kat and Maysa were working towards upgrading the API from V3 to V4.

Also the namespace is V4:
```cs
namespace cv19ResSupportV3.V4.Controllers
```

Reply:
Due to time constraints, some V4 methods were left in V3 folder, even though they were refactored into V4.